README.md

```diff
@@ -1,8 +1,6 @@
 # GeoFabrics
 
-## Build Status
-
-![Linux](https://github.com/rosepearson/GeoFabrics/actions/workflows/linux-test.yml/badge.svg)
+![Linux](https://github.com/rosepearson/GeoFabrics/actions/workflows/linux-test.yml/badge.svg)  ![Docs](https://github.com/rosepearson/GeoFabrics/actions/workflows/update-pages.yml/badge.svg)
```

Comment:
The docs badge seem to be missing, it doesn't render as an icon.

Reply:
Yes.. it's not pointing to this branch so it wouldn't - but it also didn't seem to work when I tried pinning it to the right branch. I'll remove and try add in the future.

Reply:
done